drivers/SmartThings/zigbee-window-treatment/src/mic/init.lua

```diff
@@ -0,0 +1,187 @@
+-- Copyright 2022 SmartThings
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+local log = require "log"
+local capabilities = require "st.capabilities"
+local zcl_clusters = require "st.zigbee.zcl.clusters"
+local window_shade_defaults = require "st.zigbee.defaults.windowShade_defaults"
+local WindowCovering = zcl_clusters.WindowCovering
+local cluster_base = require "st.zigbee.cluster_base"
+local utils = require "st.utils"
+local data_types = require "st.zigbee.data_types"
+local window_preset_defaults = require "st.zigbee.defaults.windowShadePreset_defaults"
+
+local micronic_window_shade_FINGERPRINTS = {{
+    mfr = "micronic-ko",
+    model = "acm301"
+}}
+
+local is_micronic_window_shade = function(opts, driver, device)
+    for _, fingerprint in ipairs(micronic_window_shade_FINGERPRINTS) do
+        if device:get_manufacturer() == fingerprint.mfr and device:get_model() == fingerprint.model then
+            return true
+        end
+    end
+    return false
+
+end
+
+local INVERT_CLUSTER = 0xFC00
+local INVERT_CLUSTER_ATTRIBUTE = 0x0000
+
+local function invert_preference_handler(device)
+
+    log.debug(string.format(" invert_preference_handler~~  (%s) invert(%s)", device.preferences, device.preferences.reverse))
+
+    local invert_value = device.preferences.reverse
+    local invert_cluster_cmd = cluster_base.write_manufacturer_specific_attribute(device, INVERT_CLUSTER,
+        INVERT_CLUSTER_ATTRIBUTE, 0x0000, data_types.Boolean, invert_value)
+    device:send(invert_cluster_cmd)
```

Comment:
this attribute is read but the response is unhandled

Reply:
should i must have to handling the response? if so, could you give me any example code or guides?

Reply:
never mind, this is a write, not a read